docs/reference/library.md

```diff
@@ -12,120 +12,163 @@ The library can also notify the ASAB microservice about changes, e.g. for automa
 
 The library content is organized in a simplified file system manner, with directories and files.
 
-Example of the library structure:
+!!! example "Example of the library structure"
 
-```
-+ /folder1/
-  - /folder1/item1.yaml
-  - /folder1/item2.json
-+ /folder2/
-  - /folder2/item3.yaml
-  + /folder2/folder2.3/
-    - /folder2/folder2.3/item4.json
-```
+	```
+	+ /folder1/
+		- /folder1/item1.yaml
+		- /folder1/item2.json
+	+ /folder2/
+		- /folder2/item3.yaml
+		+ /folder2/folder2.3/
+			- /folder2/folder2.3/item4.json
+	```
 
-## Library path rules
+!!! warning "Library path rules"
 
-- Any path must start with "/", including the root path.
-- The folder path must end with "/".
-- The item path must end with a file extension (e.g. ".json").
+	- Any path must start with "/", including the root path.
+	- The folder path must end with "/".
+	- The item path must end with a file extension (e.g. ".json").
 
 ## Layers
 
-The library content can be organized into an unlimited number of layers.
-Each layer is represented by a **provider** with a specific configuration.
+The library content can be organized into an unlimited number of **layers**.
+Each layer is represented by a **provider** (e.g. filesystem, zookeeper, git, ...) with a specific configuration. Two layers can have the same provider but different base paths.
 
 The layers of the library are like slices of Swiss cheese layered on top of each other.
 Only if there is a hole in the top layer can you see the layer that shows through underneath.
 It means that files of the upper layer overwrite files with the same path in the lower layers.
 
-The first provider is responsible for providing '/.disabled.yaml' that controls the visibility of items. 
-If '/.disabled.yaml' is not present, then is considered empty.
-
-## Library service
+<figure markdown>
+  ![Illustration of Library layers](../images/library/asab-library.drawio.png)
```

Comment:
1) The name of the file should be "asab-library-layers.drawio.png"
2) Layers are numbered from 0, with "0" having a special meaning in asap-library micro service